iron-icon.html

```diff
@@ -52,7 +52,7 @@
 
     <iron-icon icon="fruit:cherry"></iron-icon>
 
-See [iron-iconset](#iron-iconset) and [iron-iconset-svg](#iron-iconset-svg) for more information about
+See [iron-iconset](iron-iconset) and [iron-iconset-svg](iron-iconset-svg) for more information about
 how to create a custom iconset.
 
 See [iron-icons](https://elements.polymer-project.org/elements/iron-icons?view=demo:demo/index.html) for the default set of icons.
```

Comment:
That _would_ be the right link for iron-icons if we had any docs for iron-icons ... but since those docs don't show up at present, this links directly to the demo, instead.

Note that it would be more accurate to say, "see the iron-icons demo" and the link could be shortened to: `[iron-icons demo](iron-icons?view=demo:demo/index.html)`.